### ISSUE TITLE
Netflix blocked

### ISSUE THREAD

**Issue by Erroneuz** [Perflyst/PiHoleBlocklist#2]
```
# Other useless connections from Smart-TV

# Netflix
api-global.netflix.com
ichnaea.netflix.com
customerevents.netflix.com
appboot.netflix.com
```

Except that actually blocks Netflix entirely, not only from a TV, but also from eg. Windows 10 App.

**Comment by Perflyst** [comment]
Thanks for your reply. Can you tell me exactly which one needs to be released? Or do they all have to be removed? I don't have netflix, so I can't test it myself.

**Comment by Erroneuz** [comment]
I believe it's appboot.netflix.com, but let me follow up on that, I'm gonna do some testing later today.

**Comment by Perflyst** [comment]
Fixed with 32b4425d50677d19e6a15c73c092af4cff328c76
